be/src/Project/Services/Client/Client.Api/Services/Groups/GroupsService.cs

```diff
@@ -0,0 +1,213 @@
+﻿using Client.Api.Infrastructure;
+using Client.Api.Infrastructure.Groups.Entities;
+using Client.Api.ServicesModels.Groups;
+using Foundation.Extensions.Exceptions;
+using Foundation.Extensions.ExtensionMethods;
+using Foundation.GenericRepository.Extensions;
+using Foundation.GenericRepository.Paginations;
+using Foundation.Localization;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Localization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Client.Api.Services.Groups
+{
+    public class GroupsService : IGroupsService
+    {
+        private readonly ClientContext context;
+        private readonly IStringLocalizer<ClientResources> clientLocalizer;
+
+        public GroupsService(
+            ClientContext context)
+        {
+            this.context = context;
+        }
+
+        public async Task<Guid> CreateAsync(CreateGroupServiceModel model)
+        {
+            var group = new Group();
+
+            this.context.Groups.Add(group.FillCommonProperties());
+            var groupTranslation = new GroupTranslation
+            {
+                Name = model.Name,
+                Language = model.Language,
+                GroupId = group.Id
+            };
+
+            this.context.GroupTranslations.Add(groupTranslation.FillCommonProperties());
+            await this.context.SaveChangesAsync();
```

Comment:
**suggestion:** Have you implemented the language fallback as in categories, etc.?

Reply:
@dawiddworak88  Yes :)